test/test-config/test-config.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test_config
+
+import "github.com/limechain/hedera-eth-bridge-validator/config"
+
+var (
+	TestConfig = config.Config{
```

Comment:
Still not sure whether we have to instantiate `clients/repositories/services` with a real configuration. `PrivateKey:            "9f6da11eecc0fd7cb081d2aee88092ee3436397916c894ad6cd80a79009c0ded"` - is this safe?

Reply:
Those are random generated pr keys